### PR TITLE
Clarify wildcard path with regular expressions

### LINKED ISSUE #123: HTTP path operation name should include JAX-RS app path mapping
There can be two JAX-RS apps exposing endpoints mapped to the same URLs. At the moment spans for these endpoints would have the same operation name.

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #123 

@fmhwong could you please have a look?

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: spec/src/main/asciidoc/microprofile-opentracing.asciidoc [hunk 1]
```diff
@@ -135,8 +135,8 @@ The implementation has to provide two operation name providers:
 * `class-method` - the provider for the default operation name.
 * `http-path` - the operation name has the following form
 `<HTTP method>:<@Path value of endpoint's class>/<@Path value of endpoint's method>`.
-For example if the class is annotated with `@Path(service)` and method `@Path(endpoint)` then
-the operation name is `GET:/service/endpoint`.
+For example if the class is annotated with `@Path("service")` and method `@Path("endpoint\{id: \\d+}")` then
```
Comment (fmhwong) [inline spec/src/main/asciidoc/microprofile-opentracing.asciidoc]: Should `@Path("endpoint\{id: \\d+}")` be `@Path("endpoint/{id: \\d+}")`?

File: spec/src/main/asciidoc/microprofile-opentracing.asciidoc [hunk 2]
```diff
@@ -135,8 +135,8 @@ The implementation has to provide two operation name providers:
...
+the operation name is `GET:/service/endpoint\{id: \\d+}`.
```
Comment (fmhwong) [inline spec/src/main/asciidoc/microprofile-opentracing.asciidoc]: and `GET:/service/endpoint/{id: \\d+}` ?

File: spec/src/main/asciidoc/microprofile-opentracing.asciidoc (hunk 1 above)
Comment (pavolloffay) [reply spec/src/main/asciidoc/microprofile-opentracing.asciidoc]: yes there is a typo. I will fix it

**Comment by pavolloffay** [comment]
@fmhwong PR updated, could you please re-review and approve?

**Pull request merged**